internal/test/e2e/e2e_test.go

```diff
@@ -224,7 +224,7 @@ func setupLocalCluster(t *testing.T) (*cluster.Context, func()) {
 }
 
 func runAllTests(t *testing.T, kubectl *kubectlContext) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*5)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*10)
```

Comment:
I don't think this context timeout governs the entire test run.
As far as I can see it is supplied to the etcdclient API and governs the time taken for an individual ETCD API call.
In which case, 10 minutes it too much.

Reply:
Reduced back to five.